scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -99,15 +108,78 @@ private fun createScanner(): Scanner {
     )
 }
 
-private fun createAnalyzerResult(): OrtResult =
-    getAssetFile("analyzer-result.yml").readValue()
+private fun createAnalyzerResult(): OrtResult {
```

Comment:
Have you considered using (and if required, extending) `OrtResultDsl` instead? See https://github.com/oss-review-toolkit/ort/blob/3949b25cccb8a2a11eddcab7fec26d7364a740ca/evaluator/src/test/kotlin/EvaluatorTest.kt#L122 for an example use, and also see https://github.com/oss-review-toolkit/ort/issues/2842 for the larger context.

Reply:
No, I haven't - good hint. I'd prefer to stick to this approach for now and once done look into migrating it to that DSL to remain focused on my main topic.

Reply:
> once done look into migrating it to that DSL

Mind me doing that migration then ASAP? Because I'd prefer to not introduce new / temporary code with "manually crafted" test result data anymore.

Reply:
>  Mind me doing that migration then ASAP? Because I'd prefer to not introduce new / temporary code with "manually crafted" test result data anymore.

I would like to finish my work on this (which is beyond this PR) first before refactoring it. I don't mind whether you or I do it, but I do care that it's not done in the middle of my implementation work.

Reply:
> Does that mean you're planning to modify the test data further after this PR? If so, that was not clear to me.

Oh, I see. So, yes - I'm planning to make the testing way more detailed in several upcoming PRs of mine (along with improving the filtering of the date we put into `OrtResult`).

> But then especially a conversion to the DSL would make sense, as it eases modifying the test data further.

Sure, let's just see once my changes have been made how the requirements to that DSL look like.